doc/Changelog.md

```diff
@@ -1,25 +1,27 @@
 Changelog {#Changelog}
 =========
 
-## git master {master)
+<!-- Uncomment at first change # git master {#master) -->
+
+# Release 1.6.0 (2015-10-04) {#Release010600}
 
 * [#24](https://github.com/BlueBrain/Brion/pull/24):
   Add getURI() method in SpikeReport.
+* [#22](https://github.com/BlueBrain/Brion/pull/22):
+  spikeConverter can process spikes to and from stream-type SpikeReport plugins.
+* [#22](https://github.com/BlueBrain/Brion/pull/22):
+  SpikeReport DSO plugins in the LD_LIBRARY_PATH are loaded automatically.
+* [#12](https://github.com/BlueBrain/Brion/pull/12):
+  Extended brion::Synapse to also support non-merged synapse files
+* [#12](https://github.com/BlueBrain/Brion/pull/12):
+  Add brion::Target::parse() to resolve a given target name
 * [#9](https://github.com/BlueBrain/Brion/issues/9):
   Extend SWC parser to support fork and end points and undefined section points.
   The Brion::SectionType enum has not been extended to include end and fork
   points, these types are translated into the most reasonable one based on the
   point ancestors.
-* [#12](https://github.com/BlueBrain/Brion/pull/12):
-  Add brion::Target::parse() to resolve a given target name
-* [#12](https://github.com/BlueBrain/Brion/pull/12):
-  Extended brion::Synapse to also support non-merged synapse files
-* [#22](https://github.com/BlueBrain/Brion/pull/22):
-  SpikeReport DSO plugins in the LD_LIBRARY_PATH are loaded automatically.
-* [#22](https://github.com/BlueBrain/Brion/pull/22):
-  spikeConverter can process spikes to and from stream-type SpikeReport plugins.
 
-## Release 1.5.0 (2015-07-07) {changelog_1_5_0}
+# Release 1.5.0 (2015-07-07) {#Release010500}
```

Comment:
We don't need the {#targets}, they just make the github rendering ugly and add little in doxygen.